error-messages/user.messages.js

```diff
@@ -4,6 +4,7 @@ const INPUT_NOT_VALID = 'INPUT_NOT_VALID';
 const WRONG_CREDENTIALS = 'WRONG_CREDENTIALS';
 const USER_NOT_AUTHORIZED = 'USER_NOT_AUTHORIZED';
 const EMAIL_ERROR = 'EMAIL_ERROR';
+const NO_ACCESS = 'NO_ACCESS';
```

Comment:
Please, rename it to the `INVALID` permissions, it is much easier to understand, thanks

Reply:
Renamed!